addons/pos_self_order/views/qr_code.xml

```diff
@@ -21,17 +21,22 @@
     <template id="qr_codes_page">
         <t t-call="web.basic_layout">
             <div class="page">
-                <h1 class="mb-5 ms-2">
+                <h1 class="text-center mb-5">
                     QR Code Menu
                 </h1>
                 <!-- There is a very annoying problem here:
                 wkhtmltopdf does not handle flex-wrap. ( at least not in this version )
                 This makes it so the bootstrap grid system does not work.
                 We have to do a workaround -->
-                <t t-foreach="groups_of_tables" t-as="group_of_tables">
-                    <div class="d-flex">
-                        <t t-foreach="group_of_tables" t-as="table">
-                            <t t-call="pos_self_order.qr_code"/>
+                <t t-foreach="pages_of_tables" t-as="page_of_tables">
+                    <div style="break-after: page;">
+                        <t t-foreach="page_of_tables" t-as="row_of_tables">
+                        <!-- FIXME: align the qr codes -->
```

Comment:
From the print, the qr codes looked align already. Maybe no need to keep this comment? But it's your call, perhaps I'm not seeing the whole picture.